### PR TITLE
[wasm] Call the correct single step trampoline. Fixes some step-into scenarios.

### PR DESCRIPTION
**Pull request by kumpera**

### CODE REVIEW

File: mono/mini/mini-wasm-debugger.c [hunk 1]
```diff
@@ -427,6 +428,17 @@ mono_wasm_set_breakpoint (const char *assembly_name, int method_token, int il_of
 	return req->id;
 }
 
+EMSCRIPTEN_KEEPALIVE int
```
Comment (jaykrell) [inline mono/mini/mini-wasm-debugger.c]: gboolean?

Comment (kumpera) [reply mono/mini/mini-wasm-debugger.c]: Since this is called by JS, which won't get a boolean, being explicit about values makes sense?

**Comment by kumpera** [comment]
@monojenkins build OS X x64

**Comment by kumpera** [comment]
@monojenkins build Linux AArch64

**Comment by kumpera** [comment]
Pushes changes to resolve merge conflict with previous PR.

**Comment by kumpera** [comment]
@monojenkins build failed

**Comment by kumpera** [comment]
@monojenkins build Windows x64

**Comment by kumpera** [comment]
@monojenkins build Linux 64

**Pull request merged**